### PR TITLE
248k preset upgrade + add 248K+K2(without SE) preset

### PR DESCRIPTION
**Pull request by quindao**
I decided to add a 248k + K2 Preset as I started a playthrough and saw there was a lack of such a thing. I also realized that the current 248K preset had a huge materials group which I reformated and split into a group for each tier of tech in 248K.

### CODE REVIEW

File: presets/presets.lua [hunk 1]
```diff
@@ -1077,74 +1077,189 @@ presets = {
...
+            {type="item",       name="locomotive", quantity=1},
+            {type="item",       name="construction-robot", quantity=1},
+            {type="item",       name="logistic-robot", quantity=1},
+            
+            {type="group",      name="Electronics"},
+            {type="item",       name="el_materials_ALK", quantity=1},
+            {type="item",       name="el_energy_crystal_item", quantity=1},
+            {type="item",       name="el_diesel_train_item", quantity=1},
+
+            {type="group",      name="Fisson"},
```
Comment (Wiwiweb) [inline presets/presets.lua:1222]: typo

File: presets/presets.lua [hunk 2]
```diff
@@ -1077,74 +1077,189 @@ presets = {
...
+            {type="item",       name="fu_tech_sign_item", quantity=1},
+            {type="item",       name="fu_robo_logistic_item", quantity=1},
+            {type="item",       name="fu_robo_construction_item", quantity=1},
+            {type="item",       name="fu_stelar_reactor_item", quantity=1},
             {type="item",       name="fu_tokamak_reactor_item", quantity=1},
-            {type="item",       name="fu_star_engine_core_item",quantity=1},
+            {type="item",       name="fu_star_engine_core_item", quantity=1},
+
+            {type="group",      name="Exotics"},
             {type="item",       name="fu_space_probe_science_item", quantity=1},
```
Comment (Wiwiweb) [inline presets/presets.lua:1138]: Is that one meant to be in "Fusion"?

File: presets/presets.lua [hunk 3]
```diff
@@ -1077,74 +1077,189 @@ presets = {
...
+            {type="technology", name="kr-advanced-pickaxe",          quantity=1},
+            {type="item",       name="kr-intergalactic-transceiver", quantity=1},
+
+            {type="group",      name="K2 Resources"},
+            {type="fluid",      name="mineral-water",                quantity=1},
+            {type="item",       name="silicon",                      quantity=1},
+            {type="item",       name="rare-metals",                  quantity=1},
+            {type="item",       name="imersite-powder",              quantity=1},
+
+            {type="group",      name="vanilla"},
```
Comment (Wiwiweb) [inline presets/presets.lua:1211]: Can you capitalize Vanilla and also put it above the K2 categories?

File: presets/presets.lua (hunk 2 above)
Comment (quindao) [reply presets/presets.lua:1138]: no as althought its unlocked at the end of the fusion tier its used throughout the exotics tier

File: presets/presets.lua (hunk 3 above)
Comment (quindao) [reply presets/presets.lua]: done

File: presets/presets.lua (hunk 1 above)
Comment (Wiwiweb) [reply presets/presets.lua:1221]: It's "Fission" right?

Comment (quindao) [reply presets/presets.lua:1221]: yeah thanks for cathing that im blind as it turns out

Comment (quindao) [reply presets/presets.lua]: fixed

**Comment by Wiwiweb** [comment]
Thanks! 👍

**Pull request merged**